### PR TITLE
Tchsm knot server

### PR DESCRIPTION
**Pull request by fmontoto**
Add thcm-knot demo using ubuntu 14 for the server and alpine for the nodes. Provides a DNSSEC dummy zone signed using tchsm.

### CODE REVIEW

File: tchsm/demo/ubuntu14-knot/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TCHSM Demo
+
+# Demo of an DNS server with DNSSEC support using the TCHSM
```
Comment (camilog) [inline tchsm/demo/ubuntu14-knot/README.md]: "Demo of a DNS server" (a instead of an)

**Comment by camilog** [comment]
Besides of that minor typo, LGTM

**Comment by raticate** [comment]
realpath must be installed to run demo.sh

**Comment by fmontoto** [comment]
59fb73f Remove realpath as dependency and replaces it with readlink, it's supposed to be more standard.

58d2260 Fixes typo

**Comment by Zeeerooo** [comment]
LGTM

**Pull request merged**